images/capi/ansible/windows/roles/cloudbase-init/templates/cloudbase-init-unattend.conf

```
@@ -17,7 +17,7 @@ debug=true
 logdir=C:\Program Files\Cloudbase Solutions\Cloudbase-Init\log\
 logfile=cloudbase-init-unattend.log
 default_log_levels=comtypes=INFO,suds=INFO,iso8601=WARN,requests=WARN
-logging_serial_port_settings=COM1,115200,N,8
+logging_serial_port_settings={{ cloudbase_looging_serial_port }}

```

using COM1 was interfering with AWS initializing the instance. This resulted in cloudbase init failing to access the metadata server

---

might be worth commenting ? (ansible conf files iirc allow `#` comments)

---

@jayunit100 I think COM1 is used for Azure so we need to leave this in, good to have it parameterised though! 

nit @gab-satchi  
```suggestion
logging_serial_port_settings={{ cloudbase_logging_serial_port }}
```